Library/Homebrew/diagnostic.rb

```diff
@@ -785,6 +785,16 @@ def check_coretap_git_origin
               git -C "#{coretap_path}" remote set-url origin #{Formatter.url("https://github.com/Homebrew/homebrew-core.git")}
           EOS
         end
+
```

Comment:
Add a `return if ENV["CI"] || ENV["JENKINS_HOME"]` so this isn't hit on CI.

Reply:
Happy to do that. Just curious, why does it matter if CI hits this? Is it just for test coverage?

Reply:
It is because `brew doctor` should not fail on CI. PRs would fail, because they are not on `master`.